VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -636,7 +640,7 @@ def write_to_status_file(self, stat_rept_file):
         try:
             tempStatusFile =  os.path.join(self._context._status_dir, CommonVariables.TempStatusFileName)
             if self._context._status_file:
-                with open(tempStatusFile,'w+') as f:
+                with open(tempStatusFile,'w') as f:
```

Comment:
why this change?

Reply:
w+ opens the file in both read and write mode. Since we are only performing the write operation, opening in read mode is not required.